crates/orderbook/openapi.yml

```diff
@@ -617,6 +621,10 @@ components:
             For ethflow orders - order that are placed onchain with native eth -, this field
             contains a struct with two variables user_valid_to and is_refunded
           $ref: "#/components/schemas/EthflowData"
+        onchainUser:
+          description: |
+            TODO: smth related to ETH flow
```

Comment:
This represents the actual trader of an on-chain order. In that case, the `owner` would be the EthFlow contract in the case of EthFlow orders and not the actual trader.

Reply:
Not sure if that gives Alexandr enough context on what to put there. Maybe

> For ethflow orders the `owner` field is set to the ethflow contract and this field is set to the true owner of the order.

Reply:
I just copied the Nick's message :)
Thank you!